docs/MEETUP_TOPICS.md

```diff
@@ -0,0 +1,84 @@
+
+8. **Kubernetes Storage**  
+This topic will cover [types of volumes](https://kubernetes.io/docs/concepts/storage/volumes/), [persistent volumes](https://kubernetes.io/docs/concepts/storage/volumes/), what's a [persistent volume claims](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims), and [how to configure an app to use persistent volumes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-volume-storage/).
+
+9. **Troubleshooting Kubernetes**  
+Without understanding Kubernetes, troubleshooting issues in the cluster is not an easy task. This topic will give tips and what to see first when things go wrong: [troubleshoot app failure](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-application/), [troubleshoot clusterr failure](https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/). Reference: [Brandon Philips, Kubernetes Day 2: Cluster Operations](https://www.youtube.com/watch?v=U1zR0eDQRYQ).
+
+10. **Kubernetes Logging and Monitoring**  
+This topic will cover [basic logging architecture](https://kubernetes.io/docs/concepts/cluster-administration/logging/), [what to monitor](https://www.datadoghq.com/blog/monitoring-kubernetes-era/), which includes [cluster components](https://kubernetes.io/docs/tasks/debug-application-cluster/resource-usage-monitoring/). Reference: [kubectl logs](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#logs).
+
+11. **Intro to Helm**  
+Helm is a package manager for Kubernetes, like how `apt` is for `ubuntu`. This will not be in CKA exam but will be very useful, necessary, and practical to someone new to Kubernetes. Reference: [Helm docs](https://helm.sh/docs/intro/).
+
+12. **Extending Kubernetes**  
+This is the next step of Kubernetes, good for the last session of curriculum. It won't be in CKA exam but good for someone who has completed the entire core curriculum and looking to figure out what to learn next. This topic will cover [extending Kubernetes](https://kubernetes.io/docs/concepts/extend-kubernetes/extend-cluster/) with [kubectl plugins](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/), [Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/), what's [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) and [operator SDK](https://github.com/operator-framework/operator-sdk).
+
+
+#### Category #2: Coud Native Stack Exploration
```

Comment:
typo ? `Coud Native Stack Exploration` should be `Cloud Native Stack Exploration`

Reply:
fixed in https://github.com/cloudnative-id/meetups/pull/33/commits/6a920f254edebd32fa1be3b8ec949a8015650052